18F/identity-idp
# LG-6066: Implement print button on FSMv2 Personal Key step

**Why**:

- So that we can retain feature parity with the existing screen.
- To reduce size and scope of common application bundle
- To create a more rigid connection between print JavaScript functionality and server-side render logic
- To improve test coverage for existing print button behavior

**Testing Instructions:**

1. Set `idv_api_enabled: "true"` in local `config/application.yml`
2. Go to: http://localhost:3000/verify/v2/personal_key
3. Click "Print"
4. Observe that print dialog is shown

![image](https://user-images.githubusercontent.com/1779930/163400434-e3ff8dcb-5551-40fe-8bdb-398bcf192c30.png)
